docs/source/blocks/block-style-wrapper.md

```diff
@@ -25,45 +25,68 @@ See below for an example.
 
 ```js
-  // (in your block config object)
-  my_custom_block: {
-    // (more block settings)
-    enableStyling: true,
+const stylingSchema = {
+  fieldsets: [{
+    id: 'styling',
+    title: intl.formatMessage(messages.styling),
+    fields: ['styles'],
+  }],
+
+  properties: {
+    styles: {
+      widget: 'object',
+      title: intl.formatMessage(messages.styling),
+      schema: stylesSchema({
+        schema: defaultStyleSchema({ formData, intl }),
+        formData,
+        intl,
+      }),
+    }
   }
+}
 ```
 
-```{note}
-This will work if your block uses the `BlocksForm` component to define schema-driven block configuration settings.
-```
+The `addStyling` helper adds the fieldset and the `styles` field inside the `Styling` fieldset for you, so when defining your block schema you can do:
 
-This will add a new fieldset `Styling` at the end of your block schema settings with a single `styles` object field in it.
-By default, this object field has only one field: `align`. It is configured by `defaultSchema` in `src/components/manage/Blocks/Block/StylesSchema.jsx`.
+```js
+import { addStyling } from '@plone/volto/helpers/Extensions/withBlockSchemaEnhancer';
 
-## Extending the default `styles` field in `Styling` fieldset
+export const TeaserSchema = ({ intl }) => {
+  const schema = {
+  // Here your block schema
+  };
 
-You can modify the default set of styles by using a `schemaEnhancer` function in the same way that you would for any block schema enhancer.
-Use the `stylesSchema` key in your block configuration object as follows:
+  addStyling({ schema, intl });
 
-```js
-  // (in your block config object)
-  my_custom_block: {
-    // (more block settings)
-    enableStyling: true,
-    stylesSchema: myCustomStyleSchema
-  }
-```
+  // Here you add your custom styling properties to the `styles` object
+  schema.properties.styles.schema.properties.align = {
+    widget: 'align',
+    title: intl.formatMessage(messages.align),
+    actions: ['left', 'right', 'center'],
+  };
 
-```{note}
-The signature for a `schemaEnhancer` is `({schema, formData, intl})`. You can find the reference of the default schema in `@plone/volto/components/manage/Blocks/Block/StylesSchema`.
+  // and finally add it to the default fieldset
+  schema.properties.styles.schema.fieldsets[0].fields.push('align');
+
+  return schema;
+};
 ```
 
+You can add a set of style fields defining your block styles (eg. alignment, background color, etc...) in your block schema by adding them to the `styles` object field as shown above.
+
 ## The `styles` field
 
 The `styles` field is mapped to an `objectWidget`.
-The `stylesSchema` adds the fields into this field, creating an object that is the sum of all of the fields assigned to it and its values.
+This is the shape of a possible set of styles.
+The Style Wrapper will read them and inject the styles in the edit and view components as is shown in the next sections.
```

Comment:
I'm not sure why "Style Wrapper" is capitalized. It's not a proper or formal noun, is it?

```suggestion
The style wrapper will read the styles and inject them into the edit and view components as shown in the next sections.
```